app/src/main/java/ru/practicum/android/diploma/data/converter/AreaConverter.kt

```diff
@@ -0,0 +1,18 @@
+package ru.practicum.android.diploma.data.converter
+
+import ru.practicum.android.diploma.data.filter.country.dto.AreaDtoResponse
+import ru.practicum.android.diploma.domain.country.Country
+
+object AreaConverter {
+
+    fun AreaDtoResponse.mapToCountry(): Country {
+        return Country(
+            id = id,
+            parentId = parentId,
+            name = name,
+            areas = areas
+        )
+    }
```

Comment:
```suggestion
    fun AreaDtoResponse.mapToCountry() = Country(
            id = id,
            parentId = parentId,
            name = name,
            areas = areas
        )
```

Можно сокращать данные конструкции примерно вот так